### PR TITLE
#164069323 Estimating reading time

### PR DESCRIPTION
**Pull request by akram256**
**What does this PR do?**
* Estimating reading time for an article 

**Description of the tasks to be completed?**

* Implementing time it takes to read a specific article.

**How should this be manaully tested?**

Git checkout `ft-estimate-time-to-read-article-164069323` in your terminal to switch onto your branch
`Run python manage.py makemigrations`
`Run python manage.py migrate`
Then you can run your server with python manage.py migrate
**Don't forget to register, verify account then log in**
**Then copy and paste the token to headers, Authorization, choose Bearer Token**

**Get all articles**
`Get /api/articles/ to get all articles`

Screenshot 2019-03-13 at 23 32 37

[Delivers #164069323]

### CODE REVIEW

File: authors/apps/articles/models.py [hunk 1]
```diff
@@ -88,6 +91,13 @@ def average_rating(self):
         ratings = self.scores.all().aggregate(score=Avg("score"))
         return float('%.2f' % (ratings["score"] if ratings['score'] else 0))
 
+    def calculate_reading_time(self):
```
Comment (Stanley-Okwii) [inline authors/apps/articles/models.py:94]: @akram256 , I would advice using a richer library/package than this rudimentary approach. Your approach does not take into account things such as `emojis`, `markdowns` and other formatting characters.

**Review by Stanley-Okwii — CHANGES_REQUESTED** [review]
Please work this !

File: authors/apps/articles/models.py (hunk 1 above)
Comment (akram256) [reply authors/apps/articles/models.py:94]: Give me an example at least, this is what I could manage to understand

**Review by nyabongoedgar — APPROVED** [review]

File: authors/apps/articles/models.py (hunk 1 above)
Comment (akram256) [reply authors/apps/articles/models.py:94]: Worked on it

**Review by NLSanyu — APPROVED** [review]

**Review by Stanley-Okwii — APPROVED** [review]

**Review by walimike — APPROVED** [review]

**Review by PatrickCmd — APPROVED** [review]

**Pull request merged**